devices/src/legacy/uart_pl011.rs

```diff
@@ -280,13 +288,53 @@ impl Pl011 {
                     return Err(Error::DmaNotImplemented);
                 }
             }
+            UARTDEBUG => {
+                self.debug = val;
+                self.handle_debug();
+            }
             off => {
+                debug!(
+                    "go to badwrite offset, offset is {}, UARTDEBUG is {}",
```

Comment:
Can you try and improve this debug message?

Reply:
I think `debug!("PL011: Bad write offset, offset: {}", off);` makes more sense to me than the previous one.